src/Tizen.Maps/Tizen.Maps/MapView.cs

```diff
@@ -787,6 +790,7 @@ protected virtual void Dispose(bool disposing)
         /// <since_tizen> 3 </since_tizen>
         public void Dispose()
         {
+            Log.Info(string.Format("ViewReady event callback is unset"));
```

Comment:
The behavior of 'string.Format(string, params object[])' could vary based on the current user's locale settings. Replace this call in 'MapView.Dispose()' with a call to 'string.Format(IFormatProvider, string, params object[])'.